### PR TITLE
kickstart 0.2.1 (new formula)

### PR DESCRIPTION
**Pull request by bosr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/kickstart.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+[[variables]]
+name = "file_name"
+default = "myfilename"
+prompt = "File name?"
+
+[[variables]]
+name = "software_project"
+default = "kickstart"
+prompt = "Which software project is awesome?"')
```
Comment (SMillerDev) [inline Formula/kickstart.rb:33]:
```suggestion
    (testpath/"template.toml").write <<~EOS
        name = "Super basic"
        description = "A very simple template"
        kickstart_version = 1

        [[variables]]
        name = "file_name"
        default = "myfilename"
        prompt = "File name?"

        [[variables]]
        name = "software_project"
        default = "kickstart"
        prompt = "Which software project is awesome?"
    EOS
```

**Comment by bosr** [comment]
rebased on master

File: Formula/kickstart.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      name = "software_project"
+      default = "kickstart"
+      prompt = "Which software project is awesome?"
+    EOS
+
+    # Run template interpolation
+    system "#{bin}/kickstart", "--no-input", testpath.to_s
+
+    assert_predicate testpath/"myfilename.txt", :exist?
+    assert_equal "kickstart is awesome!", shell_output("cat #{testpath}/myfilename.txt")
```
Comment (carlocab) [inline Formula/kickstart.rb:40]:
```suggestion
    assert_equal "kickstart is awesome!", (testpath/"myfilename.txt").read
```
We don't need to shell out to `cat` here. You may also need `chomp`, depending on whether `kickstart` inserts `\n` into the file.

Comment (bosr) [reply Formula/kickstart.rb]: Oh cool, thanks. `chomp` was not needed.

**Review by carlocab — DISMISSED** [review]
Thanks, @bosr.

We're having some issues with bottle uploads at the moment, so this may have to wait a bit before being merged. (I'll still attempt a merge now to see, though.)

**Comment by bosr** [comment]
Thanks for the update @carlocab!

**Review by carlocab — DISMISSED** [review]

**Comment by carlocab** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/e648b856335c569a07975f5632f055ec5cec2693. Thanks, @bosr.

**Pull request closed without merging**